packages/nitro-protocol/test/contracts/ninja-nitro/AdjudicatorFactory/deployAndPayout.test.ts

```diff
@@ -54,22 +52,23 @@ const addresses = {
   ERC20: undefined,
 };
 
-const tenPayouts = {ERC20: {}};
+/*const tenPayouts = {ERC20: {}};
 const fiftyPayouts = {ERC20: {}};
 const oneHundredPayouts = {ERC20: {}};
 
-// for (let i = 0; i < 100; i++) {
-//   const destination = randomExternalDestination();
-//   addresses[i.toString()] = destination;
-//   if (i < 10) tenPayouts.ERC20[i.toString()] = 1;
-//   if (i < 50) fiftyPayouts.ERC20[i.toString()] = 1;
-//   if (i < 100) oneHundredPayouts.ERC20[i.toString()] = 1;
-// }
+for (let i = 0; i < 100; i++) {
+  const destination = randomExternalDestination();
+  addresses[i.toString()] = destination;
+  if (i < 10) tenPayouts.ERC20[i.toString()] = 1;
+  if (i < 50) fiftyPayouts.ERC20[i.toString()] = 1;
+  if (i < 100) oneHundredPayouts.ERC20[i.toString()] = 1;
+}*/
 
 // Populate wallets and participants array
 for (let i = 0; i < 3; i++) {
-  wallets[i] = Wallet.createRandom();
-  participants[i] = wallets[i].address;
+  const rWallet = Wallet.createRandom();
+  wallets.push(rWallet);
+  participants.push(rWallet.address);
```

Comment:
This change is a little bit strange, since I think you are making the (3) participants all the same as each other?

Reply:
I don't think so: each run of the loop creates a new local scope. `const rWallet` is declared, populated, and GCd three times, but each is its own invocation of `Wallet.createRandom()`

Reply:
You're right of course, my mistake.